Parser/tokenizer.c

```diff
@@ -2512,7 +2516,9 @@ tok_get_fstring_mode(struct tok_state *tok, tokenizer_mode* current_tok, struct
             // to the loop for the next iteration.
             if (peek == '{' || peek == '}') {
                 if (!current_tok->f_string_raw) {
-                    warn_invalid_escape_sequence(tok, peek);
+                    if (warn_invalid_escape_sequence(tok, peek)) {
```

Comment:
The lack of error checking was making the tokenizer progress even if the warning was treated as an error `-Werror`